changelogs/fragments/compat-jinja-2.10-test.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
This should probably be a  minor change rather than a bugfix.
```suggestion
minor_changes:
```

Reply:
sure, but I'm hoping to backport it

Reply:
We most likely will not backport this.